modules/nf-core/gatk4/denoisereadcounts/main.nf

```diff
@@ -0,0 +1,57 @@
+process GATK4_DENOISEREADCOUNTS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::gatk4=4.4.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.4.0.0--py36hdfd78af_0':
+        'biocontainers/gatk4:4.4.0.0--py36hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(counts)
+    tuple val(meta2), path(pon)
```

Comment:
```suggestion
    tuple val(meta_count), path(counts)
    tuple val(meta_pon)  , path(pon)
```
Less confusing than meta/meta2

Reply:
This is done like this to keep all meta names uniform across all modules